ui/app/partials/alerts.html

```diff
@@ -2,14 +2,20 @@
 	<fieldset>
 		<legend>Filter</legend>
 
-		<label>Receivers</label>
-		<select name="select-multi" class="width-2" ng-model="receivers" multiple="multiple"
-			ng-options="r for r in allReceivers track by r">
-		</select>
+		<section>
+			<label>Receivers</label>
+			<select name="select-multi" class="width-2" ng-model="receivers" multiple="multiple"
+				ng-options="r for r in allReceivers track by r">
+			</select>
+		</section>
+
+		<section>
+			<label class="checkbox"><input type="checkbox" ng-model="hideSilenced"> Hide silenced</label>
+		</section>
 	</fieldset>
 </form>
 
-<div id="alert-groups">
+<div id="alert-groups" ng-class="{'hide-silenced': hideSilenced}">
```

Comment:
@juliusv Hmm...I think that would hide the entire `alert-groups` node. In order to avoid the DOM / CSS approach we'd probably need to do something like this on the `alert-item` (untested)...

```js
ng-hide="alert.silenced && hideSilenced"
```

`hideSilenced` is currently a property on `AlertsCtrl`s `$scope`. So we'd need to make it available in `AlertCtrl` either via `$rootScope` or a service. Thoughts?